python/satyaml/STRATOSAT-TK1.yml

```diff
@@ -12,5 +12,6 @@ transmitters:
      baudrate: 9600
      deviation: 4800
      framing: GEOSCAN
+     frame size: 66
```

Comment:
Can you also add the same to `GEOSCAN-EDELVEIS.yml`?

Reply:
GEOSCAN-EDELVEIS.yml I think it's worth deleting this file. It's no longer relevant.

Reply:
We keep SatYAML files for decayed satellites. People can use them to decode recorded IQ data.

Reply:
Okay, I'll add that. I'll also provide you with IQ to process the new GEOSCAN satellite platform.

Reply:
You can test it on this IQ recording https://drive.google.com/file/d/1YBSFlFV70HCjQ42WSrACk6HK5Wbro2nC/view?usp=sharing This signal is made on a test model of the satellite. You can make a similar satyml file for this, just specify the frame length of 74 bytes